applications/FluidDynamicsApplication/python_scripts/apply_outlet_process.py

```diff
@@ -24,7 +25,10 @@ def __init__(self, Model, settings ):
             "value"              : 0.0,
             "interval"           : [0.0,"End"],
             "hydrostatic_outlet" : false,
-            "h_top"              : 0.0
+            "h_top"              : 0.0,
+            "outlet_inflow_contribution" : false,
```

Comment:
Are you sure you prefer false as default?

Reply:
I was wondering whether we should put it active or inactive by default... I convinced myself because this, of course, slightly affects the mass conservation, something that might confuse the user if specifically checks for this without being notified that this extra contribution is added.